packages/block-editor/src/hooks/color.js

```diff
@@ -450,14 +450,22 @@ export function ColorEdit( props ) {
 		};
 	};
 
-	const enableContrastChecking =
+	const isWebAndColorNotGradient =
 		Platform.OS === 'web' && ! gradient && ! style?.color?.gradient;
 
 	const defaultColorControls = getBlockSupport( props.name, [
 		COLOR_SUPPORT_KEY,
 		'__experimentalDefaultControls',
 	] );
 
+	const enableContrastChecking =
+		isWebAndColorNotGradient &&
+		true ===
```

Comment:
Do we need to default support to true somehow, or switch this to a `! false ===` check as currently when I test this the check doesn't work for any blocks that don't have this set, eg. paragraph block, and it might be easier to turn it off when not needed than toggle it on everywhere it is needed?

Reply:
Thanks for spotting this!!

I swear I had `! false ===` in there but...

![2022-08-22 14 14 52](https://user-images.githubusercontent.com/6458278/185838627-a99aa298-b91d-4a7a-9faf-c97b82b1e392.gif)


🤦

Reply:
Fixed with comment added. 🙇